src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -630,6 +649,38 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
       .uid(label)
       .name(label)
   }
+
+  def getAvroSink[E <: ADT with EmbeddedAvroRecord[A]: TypeInformation, A <: GenericRecord: TypeInformation](dataStream: DataStream[E]): DataStreamSink[E] = {
```

Comment:
In other source and sink code there is often a fair bit of overlap between the avro and non-avro versions, and in those cases we create an underlying `_getSink` method or `_getSource` method that abstracts away the common elements of those methods. Can we do the same here to DRY this up?

Reply:
I added avro support for all the sinks and refactored to dry the implementations up.